gpt_computer_assistant/gpt_computer_assistant.py

```diff
@@ -227,10 +227,10 @@ def input_box_send_screenshot():
         self.show()
 
     def mouseMoveEvent(self, event: QMouseEvent):
-        delta = QPoint(event.globalPos() - self.old_position)
-        if event.buttons() == Qt.LeftButton and self.title_bar.underMouse():
-            self.move(self.x() + delta.x(), self.y() + delta.y())
-            self.old_position = event.globalPos()
+        delta = event.globalPos() - self.mapToGlobal(QPoint(0, 0))
+        if event.buttons() & Qt.LeftButton and self.title_bar.underMouse():
```

Comment:
```suggestion
        if self.title_bar.underMouse() and (event.buttons() & Qt.LeftButton) :
```
often_true_statement and less_often_true_statement is faster than the contrary.

And for dependancy purpose, the overhaul proposal cannot be true if underMouse == false, meaning that the important factor of the first statement is higher than the second because of the number of time you press the left button.

Reply:
Thanks for pointing that out!